flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/typeutils/AvroSchemaConverterTest.java

```diff
@@ -124,13 +117,14 @@ public void testInvalidRawTypeAvroSchemaConversion() {
                                 .build()
                                 .toRowDataType()
                                 .getLogicalType();
-        thrown.expect(UnsupportedOperationException.class);
-        thrown.expectMessage("Unsupported to derive Schema for type: RAW");
-        AvroSchemaConverter.convertToSchema(rowType);
+
+        assertThatExceptionOfType(UnsupportedOperationException.class)
+                .isThrownBy(() -> AvroSchemaConverter.convertToSchema(rowType))
+                .withMessageStartingWith("Unsupported to derive Schema for type: RAW");
```

Comment:
Personally I'm more of a fan of `assertThatThrownBy().<assertions>` because it keeps the assertions together.